### PR TITLE
Add article model

### PR DESCRIPTION
**Pull request by philipd97**
- add Article model
- using rails console to do `CRUD` operation

### CODE REVIEW

**Comment by tlkiong** [comment]
what kind of review do you expect me to have here?

**Comment by philipd97** [comment]
😅 ahh, ok I'll merge it straight

**Comment by philipd97** [comment]
@tlkiong I added the validation for this model, is review required?

File: app/models/article.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Article < ApplicationRecord
-
+  validates :title, presence: true, length: { minimum: 6, maximum: 100 }
+  validates :description, presence: true, length: { minimum: 6, maximum: 100 }
```
Comment (tlkiong) [inline app/models/article.rb:3]: use `validates_presence_of`

**Review by tlkiong — CHANGES_REQUESTED** [review]

File: app/models/article.rb (hunk 1 above)
Comment (philipd97) [reply app/models/article.rb:3]: > use `validates_presence_of`

Is this is the preferred way when want to validates using presence? Is there any difference if uses `validates :attribute_name, presence: true` ? 

I stumble upon rails style guide recommends to use `validates` if specifying the length 

https://github.com/rubocop/rails-style-guide#new-style-validations

Comment (tlkiong) [reply app/models/article.rb:3]: validates_presence_of <= only validate presence
validates <= validates more than presence on one line

**Review by tlkiong — APPROVED** [review]

**Review by LeeRenJie — APPROVED** [review]

**Pull request merged**